drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -275,12 +277,12 @@ static void gps_thread(int dev_ptr)
 		if (!has_fix) {
 			/** If nrf_recv() blocks for more than one second, a fix
 			 *  has been obtained or the GPS has timed out. Submit
-			 *  a delayed timeout work every two seconds to make
+			 *  a delayed timeout work every five seconds to make
 			 *  sure the appropriate event is propagated upon
 			 *  a block.
 			 */
 			k_delayed_work_submit(&drv_data->timeout_work,
-					      K_SECONDS(2));
+					      K_SECONDS(5));
```

Comment:
Why the increase to five seconds here?

Reply:
2 is not sufficient. Saw several occasions where timeout event was sent. Those issues went away when increasing timeout.